README.md

```diff
@@ -45,7 +45,22 @@ just see a `signer` constant.
 ## SDK Usage
 
 The entry point is the SDK constructor, it will instantiate a new client
-connected to the API endpoint corresponding to `dev` or `prod`:
+connected to the API endpoint corresponding to `dev` (development) or `stg` (staging).
+
+### Environment
+
+#### Staging
+
+This is the recommended environment for most testing use cases.
```

Comment:
Can we extend this a little bit? Something explaining why the stg is the recommended way? If it's because we do reset the dev environment more often, I'd say that.

Reply:
i.e. "This is the recommended environment for most testing use cases, since the dev environment is more subject to blockchain resets and downtimes than the stg one."